client/scripts/views/pages/subscriptions/notificationSettings.ts

```diff
@@ -99,15 +99,21 @@ const BatchedSubscriptionRow: m.Component<IBatchedSubscriptionRowAttrs, IBatched
     vnode.state.subscriptions = vnode.attrs.subscriptions;
   },
   view: (vnode) => {
+    if (vnode.state.subscriptions[0] !== vnode.attrs.subscriptions[0]) {
+      vnode.state.subscriptions = vnode.attrs.subscriptions;
+    }
```

Comment:
What is going on here? This seems like something is off: either we shouldn't be copying attrs to state, or whatever is stored in subscriptions[0] should be handled more elegantly

Reply:
handled this fix! you were right, there was no need to copy attrs into state here.